src/app/t.const.ts

```diff
@@ -119,6 +119,7 @@ export const T = {
         'IS_AUTO_ADD_TO_BACKLOG': 'F.GITLAB.FORM.IS_AUTO_ADD_TO_BACKLOG',
         'IS_AUTO_POLL': 'F.GITLAB.FORM.IS_AUTO_POLL',
         'IS_SEARCH_ISSUES_FROM_GITLAB': 'F.GITLAB.FORM.IS_SEARCH_ISSUES_FROM_GITLAB',
+        'IS_COLSE_REOPEN': 'F.GITLAB.FORM.IS_CLOSE_REOPEN',
```

Comment:
IS_COLSE_REOPEN seems like a spelling mistake

Reply:
It is. Thank you I will fix that.